core/src/main/java/io/aiven/klaw/helpers/db/rdbms/DeleteDataJdbc.java

```diff
@@ -143,18 +143,20 @@ public String deleteSchemaRequest(int avroSchemaId, int tenantId) {
     return ApiResultStatus.SUCCESS.value;
   }
 
-  public String deleteAclRequest(int aclId, int tenantId) {
+  public String deleteAclRequest(int aclId, String userName, int tenantId) {
```

Comment:
I think we better get this username from login mailService.getUserName, so we know who is submitting the req

Reply:
Agree it's passed in from the AclControllerService from this call. 
Although I can be more explicit about it if needed.

Reply:
oh no, my bad, was thinking it was the controller service